playground/infrastructure/ci_cd.py

```diff
@@ -88,9 +88,6 @@ def _run_ci_cd(step: str, raw_sdk: str, origin: Origin, subdirs: List[str]):
     logging.info("Finish of searching Playground examples")
     logging.info("Number of found Playground examples: %s", len(examples))
 
-    examples = list(filter(lambda example: example.tag.multifile is False, examples))
-    logging.info("Number of sinlge-file Playground examples: %s", len(examples))
```

Comment:
This was duplicated elsewhere, right? For posterity, could you drop a link to that location?

Reply:
Nope, there shall be no such filter any more, since multifile is supported.
Many multifile examples still will be skipped due missing `files` tag, see the check in `find_examples`

Reply:
Ah, I see - my mistake here